global.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is this going to be the default going forward? Where all repos are going to need a global.json to build?

Reply:
Yes, this is the requirement now.

Reply:
What is the driver for this requirement? Where is it coming from? global.json is a pretty big hammer that we would like folks to use as a escape hatch for when we can't maintain backwards compatibility in the SDK and now we are going to turn this on for all our repos. This is not ideal.

Reply:
SDK packages require it to specify version. RepoToolset is an SDK package.

Reply:
@jeffkl I wasn't aware that NuGet SDKs required an CLI version in global.json. I thought it required the NuGet SDK to have a version specified. Did I misunderstand that?

Reply:
I don't think it requires CLI version in global.json, just the RepoToolset SDK version. 
I put CLI version to global.json as well since it's convenient for the build script to find it there. 
What does exactly CLI do with this CLI version in global.json?

Reply:
Roslyn decided to specify dotnet SDK version in global.json: https://github.com/dotnet/roslyn/blob/master/global.json

Reply:
NuGet SDKs do not require dot cli version.

Reply:
@jaredpar What were the reasons for including dotnet SDK in global.json in Roslyn?

Reply:
Thinking about it more -- I believe we need to specify SDK version in global.json to achieve repeatable build that doesn't depend on any SDKs installed on the machine. 

Our repos are special in that developers working with them have all sorts of versions of SDKs installed on their boxes for dogfooding and testing purposes.